Sources/PackageDescription/BuildSettings.swift

```diff
@@ -387,6 +387,11 @@ public struct SwiftSetting {
         return SwiftSetting(
           name: "interoperabilityMode", value: values, condition: condition)
     }
+
+    @available(_PackageDescription, introduced: 5.9)
+    public static var _enableBuiltinModule: SwiftSetting {
```

Comment:
I'm assuming the underscore is supposed to signify internalness here? That seems extremely unfortunate, @tshortli had some ideas around exposing SPI in `PackageDescription` and we should settle on a single way for this.